docs/OnlineEditor.mdx

```diff
@@ -0,0 +1,17 @@
+---
+id: online-editor
+title: Online Editor
+---
+
+You can also use the online editor (https://casbin.org/en/editor/) to write your Casbin model and policy in your web browser. It provides functionality such as ``syntax highlighting`` and ``code completion``, just like an IDE for a programming language.
```

Comment:
Use casbin.io editor instead

Reply:
In fact, we don't have casbin.io editor yet.

Reply:
@Selflocking the editor is just another GitHub Pages project, source code hosted in: https://github.com/casbin/casbin-editor , and served at: https://casbin.org/casbin-editor by iframe. We should also have an iframe page in v2 too to do similar. Just imagine finally `casbin.org` will be totally down.